content/install/server/caddy.es.md

```diff
@@ -0,0 +1,66 @@
++++
+date = "2017-04-15T14:39:04+02:00"
+title = "Configuración con Caddy"
+url = "setup-with-caddy"
+
+[menu.install]
+  identifier = "setup-with-caddy"
+  parent = "install_server"
+  weight = 4
++++
+
+Esta guía provee un vistazo corto a la instalación de un servidor de Drone detrás de un servidor web Caddy. Éste es un archivo de configuración de proxy en caddy:
+
+```nohighlight
+drone.mycompany.com {
+    gzip {
+        not /stream/
+    }
+    proxy / localhost:8000 {
+        websocket
+        transparent
+    }
+}
+```
+
+Debe deshabilitar la compresión gzip para datos contínuos de lo contrario las actualizaciones en vivo no serán instantáneas:
```

Comment:
IMHO "datos contínuos" doesn't technically honor the original meaning of the word. Maybe use the english term here?

For "actualizaciones en vivo" I would do "actualizaciones en tiempo real".